DemoApp/DemoApp/Views/Components/RadioGroupView/RadioGroupView.swift

```diff
@@ -0,0 +1,56 @@
+import SATSCore
+import SwiftUI
+
+struct RadioGroupView: View {
```

Comment:
I would call this `RadioGroupDemoView` to be clearer what this is about

Reply:
Sure thing